packages/e2e-test-utils-playwright/src/metrics/index.ts

```diff
@@ -108,4 +146,132 @@ export class Metrics {
 				} )
 		);
 	}
+
+	/**
+	 * Returns the loading durations using the Navigation Timing API. All the
+	 * durations exclude the server response time.
+	 */
+	async getLoadingDurations(): Promise< LoadingDurations > {
```

Comment:
I'm curious to know whether playwright automatically infers the return type of `evaluate` here so that we don't have to explicitly cast it as it might be out-of-sync. 🤔

Reply:
Good call. Looks like there were more places where we cast unnecessarily. Addressed in https://github.com/WordPress/gutenberg/pull/53808/commits/33a78ebaa7521edca5f5b68c5cda0db14ed22210

Reply:
Removed a little bit too much, reverted some lines in https://github.com/WordPress/gutenberg/pull/53808/commits/71a1d581100e0348b5323fdd615c5095acaf3461 😅